graphsync.go

```diff
@@ -126,17 +126,30 @@ func (e RequestNotFoundErr) Error() string {
 	return "request not found"
 }
 
-// RemoteMissingBlockErr indicates that the remote peer was missing a block
-// in the selector requested. It is a non-terminal error in the error stream
+// MissingBlockErr indicates that the remote peer was missing a block
+// in the selector requested, and we also don't have it locally.
+// It is a non-terminal error in the error stream
 // for a request and does NOT cause a request to fail completely
-type RemoteMissingBlockErr struct {
```

Comment:
why the rename? it's still for remotes and we have other `Remote*` errors like the one you introduced below